crates/bevy_ecs/src/entity/mod.rs

```diff
@@ -1,3 +1,27 @@
+//! Entity handling types.
+//!
+//! In Bevy ECS, there is no monolithic data structure for an entity. Instead, the [`Entity`]
+//! `struct` is just an identification number that [`Component`s](crate::component::Component)
+//! can refer to in order to attach meaningful data to it. As the
+//! [book](#) explains, this is a fundamental design choice that has been made to increase
+//! performance and usability.
+//!
+//! # Usage
+//!
+//! In the following list there are links to the methods used to perform common operations
+//! involving entities:
+//!
+//! - **Spawning an empty entity:** use [`Commands::spawn`](crate::system::Commands::spawn).
```

Comment:
I think the ordering could be improved here. I would do:
- spawn
- despawn
- insert
- insert_bundle
- remove